test/integration/02-user-wallet/02-tx-onchain-fees.spec.ts

```diff
@@ -0,0 +1,84 @@
+import * as Wallets from "@app/wallets"
+import { getOnChainWalletConfig } from "@config/app"
+import { InsufficientBalanceError, LessThanDustThresholdError } from "@domain/errors"
+import { bitcoindClient, bitcoindOutside, getUserWallet } from "test/helpers"
+
+jest.mock("@services/realtime-price", () => require("test/mocks/realtime-price"))
+jest.mock("@services/phone-provider", () => require("test/mocks/phone-provider"))
+
+const defaultAmount = 6000 as Satoshis
+const defaultTarget = 3 as TargetConfirmations
```

Comment:
I wonder if we need a similar `toTargetConfs` method here as `toSats`. I'm not sure what our heuristic is for using a method vs. just using `as` to cast the type (cc @bodymindarts)

Would be separate from the already implemented `checkedToTargetConfs` which I believe is only meant to validate value coming from user input.